src/save.ts

```diff
@@ -75,9 +75,20 @@ async function run() : Promise<void> {
     // A failure to save cache shouldn't prevent the entire CI run from
     // failing, so do not call setFailed() here.
     core.warning(`Saving cache failed: ${error}`);
```

Comment:
According to this comment, should I return 0 even if the caching fails

Reply:
Yes, I think so, this change was introduced in https://github.com/hendrikmuhs/ccache-action/pull/59